dali/pipeline/operators/reader/sequence_reader_op.cc

```diff
@@ -30,17 +30,42 @@ DALI_REGISTER_OPERATOR(SequenceReader, SequenceReader, CPU);
 
 DALI_SCHEMA(SequenceReader)
     .DocStr(
-        "Read [Frame] sequences from a directory representing collection of "
-        "streams")
+        R"code(Read [Frame] sequences from a directory representing collection of streams.
+Expects file_root to contain set of directories, each of them represents one extracted video
+stream. Extracted video stream is represented by one file for each frame, sorting the paths to
+frames lexicographically should give the original order of frames.
+Sequences do not cross stream boundary and only full sequences are considered - there is no padding.
+Example:
+> file_root
+  > 0
+    > 00001.png
+    > 00002.png
+    > 00003.png
+    > 00004.png
+    > 00005.png
+    > 00006.png
+    ....
+  > 1
+    > 00001.png
+    > 00002.png
+    > 00003.png
+    > 00004.png
+    > 00005.png
+    > 00006.png
+    ....)code")
     .NumInput(0)
     .NumOutput(1)  // ([Frames])
     .AddArg("file_root",
             R"code(Path to a directory containing streams (directories representing streams).)code",
             DALI_STRING)
     .AddArg("sequence_length",
             R"code(Lenght of sequence to load for each sample)code", DALI_INT32)
+    .AddOptionalArg("step",
```

Comment:
We should agree on some common interface with `VideoReader` here.

I set the default `step` as -1, that indicates to the reader that we want the `step` to be actually be `sequence_length`.
I think that this is the intuitive default behaviour that you would expect for a sequence/video reader.

By the way I named it `count` in `VideoReader` but I will rename it `sequence_length` too for consistency, and because its more explicit. I'll add it in #360 changes.

Reply:
@szalpal what's your opinion about that?

Reply:
I agree, that most intuitive is `setp == sequence_length` for default values. Too bad we can't set it directly :(

Reply:
It's a really use-case specific feature, so I think that the
```
if (step_ == -1)
  step_ = sequence_length
```
will suffice here

Reply:
For me, more intuitive is `step = 1`, as it creates all possible sequences of given length.

Reply:
We decided to go with `step=1`.

Reply:
Agreed